src/Calendar.js

```diff
@@ -192,10 +192,10 @@ class Calendar extends Component {
     // Current month's days
     for (let i = 1; i <= dayCount; i++) {
       const dayMoment  = shownDate.clone().date(i);
-      // set days before today to isPassive
-      var _today = moment()
-      if (disableDaysBeforeToday && Number(dayMoment.diff(_today,"days")) <= -1) {
-        days.push({ dayMoment ,isPassive:true});
+      const dayMomentFormatted = dayMoment.format('YYYY-MM-DD');
+
+      if (disableDaysBeforeDate && (dayMomentFormatted < disableDaysBeforeDate)) {
```

Comment:
Since we're using lexicographic string comparison here, should we add some prop type validation to verify, that `disableDaysBeforeDate` - is passed in correct format?